hr_switzerland/models/hr_autolog.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Stephane Eicher <seicher@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+import logging
+
+from odoo import models, api, _
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class asterisk_autolog(models.Model):
```

Comment:
You can define it as `AbstractModel`, since there are no fields in this model. This way, it won't create a database table. But I would also say it could be possible as well to simply move the `agent_connexion` method in the `res.users` object, as a multi method. Then you could simply call `user.agent_connexion()`